plugins/scaffolder/src/components/MultistepJsonForm/MultistepJsonForm.tsx

```diff
@@ -30,7 +30,7 @@ import { Theme as MuiTheme } from '@rjsf/material-ui';
 import React, { useState } from 'react';
 
 const Form = withTheme(MuiTheme);
-type Step = {
+type StepType = {
```

Comment:
really is super unfortunate to add Type to types, that kind of naming should be avoided - where does this colide?